src/__tests__/reducers/userReducer.test.js

```diff
@@ -0,0 +1,39 @@
+import useReducer from '../../reducers/userReducer';
+import initialState from '../../store/initialStates/userInitialState';
+import { userActionsTypes } from '../../actions-types';
+import user from '../../__mocks__/user';
+
+describe('User reducers', () => {
+  test('GET_PROFILE', () => {
+    const reducer = useReducer(initialState, {
+      type: userActionsTypes.GET_USER,
+      payload: { ...user }
+    });
+    expect(reducer).toHaveProperty('firstName');
+    expect(reducer).toHaveProperty('lastName');
+    expect(reducer).toHaveProperty('username');
+    expect(reducer).toHaveProperty('email');
+  });
+
+  test('SIGNUP_USER', () => {
+    const reducer = useReducer(initialState, {
+      type: userActionsTypes.SIGNUP_USER,
+      payload: {}
```

Comment:
it would be better if you put some data in your payload, I can see it doesn't contain any

Reply:
well noted